app/src/main/java/com/keylesspalace/tusky/MainActivity.java

```diff
@@ -82,6 +91,11 @@
     private static final long DRAWER_ITEM_SAVED_TOOT = 9;
     private static final long DRAWER_ITEM_LISTS = 10;
 
+    @Inject
+    public MastodonApi mastodonApi;
+    @Inject
+    public DispatchingAndroidInjector<Fragment> fragmentInjector;
+
```

Comment:
the comment in line 111 is no longer true (github only lets me comment changed lines 🙄)